packages/apps/graph/src/devnet/crosschain-transfer.ts

```diff
@@ -130,7 +130,7 @@ export async function crossChainTransfer({
   );
 
   const continuation = {
-    pactId: status.continuation?.pactId,
+    pactId: status.continuation?.pactId || '',
```

Comment:
As we talked yesterday maybe we should discuss if we should change our functions to use `client-utils`

Reply:
What do we check here? Is `0` an option here? That will evaluate to `''`